ludwig/models/modules/h3_encoders.py

```diff
@@ -330,13 +319,13 @@ def __init__(
             dropout=dropout_rate,
             initializer=weights_initializer,
             regularize=weights_regularizer,
-            reduce_output=None,
+            reduce_output='None',
```

Comment:
I believe `None` should work too, doesn't it?

Reply:
Yeah it works with `None` (not as a string)...the reason I changed it was PyCharm was flagging it a warning.  Here is the warning I saw
<img width="649" alt="Screen Shot 2020-06-12 at 20 42 28" src="https://user-images.githubusercontent.com/1425269/84555880-52869600-aced-11ea-9284-2da9c40d3115.png">
I can put it back to the `None` type

Reply:
It uses `reduce_sequence` behind the sceens, which work with "none", "None" and None. But it's ok to use the string too, maybe in the future I can add more type annotations to avoid these wrong type inferences from the ide.